src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -171,20 +176,17 @@ export default {
 
 <style module="$s">
 .Button {
-	--button-large: 64px;
```

Comment:
Did Design say to remove the large button style on mobile? If not, then why did you do this? Please get alignment with Design before you make any design changes.

Reply:
Yes